cdp_scrapers/instances/kingcounty-static.json

```diff
@@ -0,0 +1,166 @@
+                "electoral_type": null,
+                "image_uri": "https://uploads.visitseattle.org/2015/03/QueenAnne_LempelZiv-720x313.jpg",
+                "external_source_id": null,
+                "roles": null
+            },
+            "external_source_id": null
+        },
+        "Dave Upthegrove": {
+            "name": "Dave Upthegrove",
+            "is_active": true,
+            "router_string": null,
+            "email": "dave.upthegrove@kingcounty.gov",
+            "phone": "206-477-1005",
+            "website": "https://kingcounty.gov/council/upthegrove.aspx",
+            "picture_uri": "https://kingcounty.gov/~/media/council/images/2018/Upthegove_2x3_print.ashx?la=en",
+            "seat": {
+                "name": "Position 5",
+                "electoral_area": "Kent, Renton",
+                "electoral_type": null,
+                "image_uri": "https://assets.atlasobscura.com/media/W1siZiIsInVwbG9hZHMvcGxhY2VfaW1hZ2VzL2ViYTUzOWViLTQ4YWQtNGJkMi04ODU1LWE0OGUxMTJmZjYwZWRmM2FmMjRjNDFjOWYwY2YzMV9JTUdfNjI3MC5KUEciXSxbInAiLCJ0aHVtYiIsIjEyMDB4PiJdLFsicCIsImNvbnZlcnQiLCItcXVhbGl0eSA4MSAtYXV0by1vcmllbnQiXV0/IMG_6270.JPG",
+                "external_source_id": null,
+                "roles": null
+            },
+            "external_source_id": null
+        },
+        "Claudia Balducci": {
+            "name": "Claudia Balducci",
+            "is_active": true,
+            "router_string": null,
+            "email": "claudia.balducci@kingcounty.gov",
+            "phone": "206-477-1006",
+            "website": "https://kingcounty.gov/council/balducci.aspx",
+            "picture_uri": "https://kingcounty.gov/~/media/council/balducci/images/2021/Balducci_highres.ashx?la=en",
+            "seat": {
+                "name": "Position 6",
+                "electoral_area": "Bellevue, Kirkland",
+                "electoral_type": null,
+                "image_uri": "https://bellevuewa.gov/sites/default/files/styles/header_image/public/media/header_image/2019-06/dtp-1600x400.jpg",
+                "external_source_id": null,
+                "roles": null
+            },
+            "external_source_id": null
+        },
+        "Pete von Reichbauer": {
+            "name": "Pete von Reichbauer",
+            "is_active": true,
+            "router_string": null,
+            "email": "pete.vonreichbauer@kingcounty.gov",
+            "phone": "206-477-1007",
+            "website": "https://kingcounty.gov/vonreichbauer",
+            "picture_uri": "https://kingcounty.gov/~/media/council/images/2018/von_Reichbauer_2x3_print.ashx?la=en",
+            "seat": {
+                "name": "Position 7",
+                "electoral_area": "Auburn, Federal Way",
+                "electoral_type": null,
+                "image_uri": "https://images.squarespace-cdn.com/content/v1/6052a75d659a54382fdd9d0c/de431180-5279-4e79-b9b8-9ac0d7ce397d/119676064_3347199895369413_1933009335190743261_o.jpg",
```

Comment:
this file has a watermark on it / text on it, not sure if we want to use that.

how aboiut this: https://i.pinimg.com/originals/7c/f0/62/7cf06219cd2967428d796bcba22313d5.jpg

Reply:
Doh sorry, I missed the watermark. I like it!